### ISSUE TITLE
Unable to have shared YAML config recoginized by plugin

### ISSUE THREAD

**Issue by wegry** [prettier/prettier-vscode#965]
### Github Repository to Reproduce Issue
Link to a Github repo that can be used to reproduce the issue.
A minimal reproduction can be [found here](https://github.com/wegry/no-shared-yaml-config).

### Steps To Reproduce:

1. Set package.json `prettier` key to be `@company-name/prettier-config`.
1. Reload vscode window.
1. Receive failure message in plugin output of vscode.

```
9/23/2019, 1:25:52 PM:
----------------------
Failed to resolve config for ~/no-shared-yaml-config/index.js. Falling back to the default config settings.
```
### Expected result

No semi should be respected.

### Actual result

It looks like the default config is being used.

### Additional information

_Feel free to attach a screenshot_.

**VS Code Version**:

**Prettier & Prettier Plugin Version**:

Prettier version: I've tried with 1.18.2 and without prettier installed locally. The repro repo doesn't have prettier installed.

Plugin version: 2.2.2

**OS and version**:

Ubuntu 18.04.

**Comment by ntotten** [comment]
After investigating this, it seems this is an issue in prettier itself rather than the plugin. I tried to get this working on version 3.0 of the plugin which just uses the built-in prettier resolver for config files.

I also ran `npx prettier` in the repo to just try to use this with prettier itself and got the following error: 

```
➜  no-shared-yaml-config git:(master) ✗ npx prettier --write ./index.js                              
[error] Invalid configuration file: Cannot find module '@company-name/prettier-config' from '/Users/ntotten/dev/no-shared-yaml-config'
```

Was this working before? Is there a way you have got it working directly from the command line?
